src/Api/Consumers.php

```diff
@@ -51,4 +51,21 @@ public function addACL($identifier, $group)
             'group' => $group,
         ])->json();
     }
+
+    /**
+     * @param string $group
+     * @return void
+     */
+    public function deleteACL($identifier, $group)
+    {
+        $this->delete("consumers/{$identifier}/acls/$group")->json();
```

Comment:
Just a small comment to add curly braces around $group to follow the convention and we're good :-)